### DISCUSSION TITLE
Is it possible to convert or use this model in fastsdcpu

### DISCUSSION THREAD

**Discussion by mikeshrimp1212** [rupeshs/fastsdcpu#188] [Q&A] [answered]
Hi guys, 
I am very noob for this 
Is it possible to convert models like [majicMIX realistic 麦橘写实](https://civitai.com/models/43331/majicmix-realistic) to work with fastsdcpu or add it directly.

**Comment by rupeshs** [comment, marked as answer]
As of now , FastSD CPU supports the model in diffuser format , also check this PR https://github.com/rupeshs/fastsdcpu/pull/163
